business-registry-dashboard/app/components/Table/AffiliatedEntity/index.vue

```diff
@@ -137,20 +170,32 @@ watch(
           @clear="affStore.affiliations.filters.type = ''"
         >
           <USelectMenu
-            v-slot="{ open }"
             v-model="affStore.affiliations.filters.type"
             :options="affStore.typeOptions"
+            selected-icon="hidden"
             :ui="{ trigger: 'flex items-center w-full h-[42px]' }"
           >
-            <UButton
-              variant="select_menu_trigger"
-              class="flex-1 justify-between text-gray-700"
-              :aria-label="$t('table.affiliation.filter.legalType.aria', { filter: affStore.affiliations.filters.type || $t('words.none') })"
-            >
-              {{ affStore.affiliations.filters.type !== '' ? affStore.affiliations.filters.type : $t('table.affiliation.filter.legalType.placeholder') }}
+            <template #default="{ open }">
+              <UButton
```

Comment:
Same thing for the 2 filters select menus, example:
![image](https://github.com/user-attachments/assets/6b49ed77-1851-4f8f-ae44-45613b04fe1c)